src/main/java/it/gov/pagopa/pu/organization/model/DistinctCollectionReasonDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.organization.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+@Entity
+@Table(name = "taxonomy")
+public class DistinctCollectionReasonDTO {
```

Comment:
The fact that is a Distinct is a query detail, not of the DTO
```suggestion
public class TaxonomyCollectionReasonDTO {
```